scripts/env/select_venv.py

```
@@ -0,0 +1,65 @@
+    return parser.parse_args()
+
+
+def create_venv(path: Path) -> None:
+    path.parent.mkdir(parents=True, exist_ok=True)
+    builder = venv.EnvBuilder(with_pip=True)
+    builder.create(path)
+
+
+def bootstrap_pip(path: Path) -> None:
+    python_path = path / ("Scripts/python.exe" if os.name == "nt" else "bin/python")
+    subprocess.run(
```

Ruff (0.15.9)

[error] `subprocess` call: check for execution of untrusted input

(S603)